dipper/sources/OMIM.py

```diff
@@ -1007,7 +1018,9 @@ def _get_pubs(self, entry, graph):
 
                 # add is_about for the pub
                 omim_id = 'OMIM:' + str(entry_num)
-                graph.addTriple(omim_id, self.globaltt['mentions'], pub_id)
+                graph.addTriple(omim_id, self.globaltt['mentions'], pub_id,
+                                subject_category=blv.Disease.value,
```

Comment:
Are we certain these are always diseases?  We may want to check their type

Reply:
@kshefchek thanks, [this commit](https://github.com/monarch-initiative/dipper/pull/746/commits/8227decaa71706b5fef66c3b561a523a2e8a270a) adds a fxn to check OMIM type using the OMIM prefix per [their faq](https://www.omim.org/help/faq#1_3)

Reply:
For consistency, could you use the code in the parent class? https://github.com/monarch-initiative/dipper/blob/2971e16/dipper/sources/OMIMSource.py#L94